coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -58,6 +60,8 @@ def __init__(self, language: str, docstyle: str, markers: (Iterable, str),
                                  'actually {}).'.format(length))
 
         self._metadata = metadata
+        self._top_padding = None if padding == () else int(padding[0])
+        self._bottom_padding = None if padding == () else int(padding[1])
 
```

Comment:
@jayvdb This answers your question :) . `.coalang` will have official standards defined.
By default `top_padding` and `bottom_padding` of DocComment are initialized with zero. If we don't have style defined. It will have default values. Also `padding`'s major motive was if a developer doesn't wants to follow official standard and decides suppose I want to have 2 blank lines at the bottom of docstring. He will set those padding variables and it will be amended by the bear part.